### ISSUE TITLE
"Server returned HTTP response code: 403" on importing sbt project to IntelliJ behind proxy

### ISSUE THREAD

**Issue by eed3si9n** [coursier/coursier#1961]
Ref https://github.com/coursier/coursier/issues/1541

This was originally reported by @texasbruce as https://github.com/sbt/sbt/issues/6300, as a resubmission of the discussion in https://github.com/sbt/sbt/issues/5501

## steps

1. Make sure IntelliJ SBT plugin has checked option "Download Library sources"
2. Import a project with any dependency that does not have source jar in the artifact repo and only binary jar
3. Import fails with 

```
[error] (updateClassifiers) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
[error] https://xxxxx/xxxx/xxxxx/xxxxxxx-sources.jar: download error: Caught java.io.IOException: Server returned HTTP response code: 403 for URL: https://xxxxx/xxxx/xxxxx/xxxxxxx-sources.jar (Server returned HTTP response code: 403 for URL: https://xxxxx/xxxx/xxxxx/xxxxxxx-sources.jar) while downloading https://xxxxx/xxxx/xxxxx/xxxxxxx-sources.jar
[error] Total time: 33 s, completed 8-Feb-2021 11:26:33 AM
```

## problem

One dependency without source shouldn't make the whole project import fails. 

## expectation

Source of dependency without source jar should be skipped and project should be imported successfully in IntelliJ

## notes

I (@eed3si9n) implemented  `missingOk` in https://github.com/coursier/sbt-coursier/pull/212, and later attempted to fix fetch error (source file that doesn't exist) in https://github.com/coursier/sbt-coursier/pull/224, but I closed it in favor of https://github.com/coursier/sbt-coursier/pull/232.

In https://github.com/coursier/coursier/issues/1541#issuecomment-628755175 Alex wrote:

> FYI everyone, coursier considers artifacts from Maven repositories as optional by default. That means if a javadoc or source JAR is missing (with a 404 status), `updateClassifiers` will not fail. There's no need for `missingOk` in most cases thanks to that.

I wonder if it's to do with 404 vs 403.

**Comment by texasbruce** [comment]
It seems have been merged in 1.5.0-RC2

**Comment by eed3si9n** [comment]
@texasbruce If you don't mind, could you confirm if the fix worked? Thnx.
